### ISSUE TITLE
Add example for how to specify a python environment to help dialog

### ISSUE THREAD

**Issue by mlangkabel** [CoatiSoftware/SourcetrailBugTracker#696]
The example could look like this:

If you would run:
```
$ cd /Users/me/miniconda3/envs
$ virtualenv py37
```

you would set it to `/Users/me/miniconda3/envs/py37`.

**Comment by thomasaarholt** [comment]
(Hi from reddit)

Just suggesting that you add a reference to `conda activate` as a comment next to the `virtualenv` as well. That's the command that conda users would likely be using to activate their environment.

**Comment by mlangkabel** [comment]
@thomasaarholt: thanks for the hint!

**Comment by thomasaarholt** [comment]
@mlangkabel The path you suggested does not appear to be working (or I am not using it right).
(sorry for slightly long post now, I was typing as I was testing)

I'm getting an error of not being able to import the package `dask`.
![image](https://user-images.githubusercontent.com/2721423/58397117-e4beec00-804f-11e9-8c1f-7f4cc7185074.png)

In my "base" python environment I do not have `dask` installed. In the `py37` environment that I have installed, `dask` is installed, and in Sourcetrail I have specified the path to this environment.

I get the above error when setting the environment path to `/Users/thomasaar/miniconda3/envs/py37`, and also when setting a "bogus" path like `/Users/me/Documents`. This makes me believe that the path you've suggested is either incorrect or not used by Sourcetrail (instead defaulting to "default python".

What does make it work, however, is including /bin at the end, which I guess is because Sourcetrail wants the path to the **python interpreter**, not the environment directory? 
Works: `/Users/thomasaar/miniconda3/envs/py37/bin`

To make it general, maybe suggest that users type `which python` (mac / linux) or `where python` (windows?) and enter the path up to python that is returned by that command.

**Comment by mlangkabel** [comment]
Can you please start Sourcetrail via console? That way you will be able to read the entire console output. If you do this, please just start the indexing for a single python file, so that it doesn't take too long. In the console output, right at the beginning of the indexing it should print which python environment is used. Does it show an error if you specify your custom environment?

**Comment by thomasaarholt** [comment]
Yep, it appears to work if (and only if) I paste the path to the python executable: `/Users/thomas/miniconda3/envs/py37/bin/python`

**Comment by thomasaarholt** [comment]
Thanks!
